src/components/views/messages/MessageActionBar.tsx

```diff
@@ -51,10 +51,22 @@ interface IOptionsButtonProps {
     getReplyChain: () => ReplyChain;
     permalinkCreator: RoomPermalinkCreator;
     onFocusChange: (menuDisplayed: boolean) => void;
+    getRelationsForEvent?: (
+            eventId: string,
+            relationType: string,
+            eventType: string
+        ) => Relations;
 }
 
 const OptionsButton: React.FC<IOptionsButtonProps> =
-    ({ mxEvent, getTile, getReplyChain, permalinkCreator, onFocusChange }) => {
+    ({
+        mxEvent,
+        getTile,
+        getReplyChain,
+        permalinkCreator,
+        onFocusChange,
+        getRelationsForEvent,
+    }) => {
         const [menuDisplayed, button, openMenu, closeMenu] = useContextMenu();
```

Comment:
this indentation feels wrong

Reply:
Is this better?

```
const OptionsButton: React.FC<IOptionsButtonProps> =
    ({
        mxEvent, getTile, getReplyChain, permalinkCreator, onFocusChange,
        getRelationsForEvent,
    }) => {
        const [menuDisplayed, button, openMenu, closeMenu] = useContextMenu();
```

Or what would you prefer?

Reply:
IMO the `const [menuDisplayed, button, openMenu, closeMenu] = useContextMenu();` should only be one-level indented from the `const OptionsButton: React.FC<IOptionsButtonProps>`

We normally do 

```
const OptionsButton: React.FC<IOptionsButtonProps> = ({
    prop,
    ...
}) => {
    code
    ...
};

Reply:
Ah yeah I prefer that too.

Reply:
Done in 9e8786cb6e.

This should be ready for re-review, thank you.